auth-web/src/util/constants.ts

```diff
@@ -204,29 +202,29 @@ export enum NrTargetTypes {
 
 export enum NrEntityType {
     // BC Entity Types:
-    FR = 'FR',
+    BC = 'BC', // Benefit Company
+    CC = 'CC', // Community Contribution Company
+    CP = 'CP', // Cooperative Association
+    CR = 'CR', // BC Limited Company
```

Comment:
I added the new types and sorted the keys (easier to see what's already there or missing).